tests/Fixtures/inc/Engine/Optimization/DeferJS/Subscriber/deferInlineJs.php

```diff
@@ -143,4 +143,126 @@ function newContent() {
 		'html'     => $html,
 		'expected' => $expected,
 	],
+
+	'testShouldExcludeUsingStringFilter' => [
+		'config' => [
+			'rocket_defer_inline_exclusions_filter' => 'first_string|third_string',
```

Comment:
What about adding a Bool (true / false), Int, Float / double, Null exclusion?
Should we add these types to fixtures?